src/app.js

```diff
@@ -29,6 +29,7 @@ class App extends Component {
   constructor() {
     super()
 
+    console.log(process.env)
```

Comment:
Hmm I think we should probably remove this haha

Reply:
Oh lord